app/controllers/chargeback_controller.rb

```diff
@@ -247,6 +248,9 @@ def cb_rate_edit
                   detail.group = r[:group]
                   detail.per_unit = r[:per_unit]
                   detail.metric = r[:metric]
+                  # Copy the measure id of the rate_detail linkig with the rate_detail_measure
+                  id_measure = ChargebackRateDetailMeasure.find_by(:name => r[:measure]).id
```

Comment:
Hi @Fryguy. I'm not sure I understand your question. 
Some ChargebackRateDetails (Allocated CPU Count, Fixed Compute Cost 1, Fixed Compute Cost 2 ...) don't have measurements associated that is why their reference (chargeback_rate_detail_measure_id) can be nil.